content-src/components/Search/Search.js

```diff
@@ -270,7 +271,7 @@ const Search = React.createClass({
       <div className="search-container" role="presentation" hidden={!this.getDropdownVisible()}>
       <section className="search-title" hidden={!formHistory.concat(suggestions).length}>
         <img id="current-engine-icon" src={currentEngine.iconBuffer} alt={currentEngine.name.charAt(0)} width="16px" height="16px" />
-        {this.props.searchHeader.replace("%S", currentEngine.name)}
+        {this.props.intl.formatMessage({id: "search_header"}).replace("%S", currentEngine.name)}
```

Comment:
Not sure if we want to maintain consistency with another format for these strings, but couldn't you just use variables for this?